specification/storage/Microsoft.BlobStorage/routes.tsp

```diff
@@ -717,8 +702,6 @@ namespace Storage.Blob {
         ...ImmutabilityPolicyExpiresOnResponseHeader;
         ...ImmutabilityPolicyModeResponseHeader;
         ...LegalHoldResponseHeader;
-        ...StructuredBodyTypeResponseHeader;
-        ...StructuredContentLengthResponseHeader;
```

Comment:
`download` blob API 
response headers changes 3

I still don't fully understand this portion, we can ask Joel once we get closer to a finished state on this proposed changes. If I had to guess, one of them is on the literal download response, the other one is in the body which is a part of the download response (the body response in the download response contained a dupe?)

Reply:
The first response is to handle the HTTP 200 case (if no status code is specified, 200 is implied).

The second response is to handle the HTTP 206 case for downloading a specified range. Note the `@statusCode statusCode: 206` on line 672.